core/src/rpc.rs

```diff
@@ -80,10 +106,15 @@ impl JsonRpcRequestProcessor {
         }
     }
 
-    pub fn get_account_info(&self, pubkey: &Pubkey) -> Result<Account> {
-        self.bank()
-            .get_account(&pubkey)
-            .ok_or_else(Error::invalid_request)
+    pub fn get_account_info(&self, pubkey: Result<Pubkey>) -> RpcResponse<Account> {
+        let bank = &*self.bank();
+        match pubkey {
+            Ok(key) => Response::new(
+                bank,
+                bank.get_account(&key).ok_or_else(Error::invalid_request),
```

Comment:
This `ok_or_else(Error::invalid_request)` is a little clunky.  We can instead return `Option<Account>`, so a missing account returns as `value: null`